### ISSUE TITLE
Implement parallel methods for errorlocating

### ISSUE THREAD

**Issue by edwindj** [data-cleaning/errorlocate#18]
To speed up the computation, it would be beneficial to check the records in parallel.

**Comment by markvanderloo** [comment]
Great idea! We need to think about the scheduling because the timings are very skewly distributed.

**Comment by edwindj** [comment]
You are right: we might take a look at the parallel LB functions (load balancing): in stead of splitting the data in front, they assign a new data item to a node when it becomes available. Down side is that the communication overhead is bigger (but we have to find out).

**Comment by edwindj** [comment]
subtask: add + operator for errorlocation objects that simplifies adding/concatenating results from locate_errors.

**Comment by markvanderloo** [comment]
I think you can just turn on `.scheduling="dynamic"`, see `?parLapply`. Would be nice to have some benchmarks. I expect it is (a lot) better, at least on average, then static scheduling.

**Comment by edwindj** [comment]
almost finished: only documentation
